azurelinuxagent/common/conf.py

```diff
@@ -243,6 +243,10 @@ def get_ext_log_dir(conf=__conf__):
     return conf.get("Extension.LogDir", "/var/log/azure")
 
 
+def get_agent_log_file():
```

Comment:
Just a bit concerned about putting this here since its not part of the conf (Maybe osutil would be a better place since the other PR eventually plans to move it there).

Reply:
I still have to revisit the logic in the other PR, and if the ultimate goal is to expose the agent log parameter in the config file, this would be the correct place. Of course, this can change, so for now this I settled for this solution.